apps/meteor/ee/app/livechat-enterprise/server/api/lib/contacts.ts

```diff
@@ -0,0 +1,52 @@
+import type { ILivechatContact, IUser } from '@rocket.chat/core-typings';
+import { License } from '@rocket.chat/license';
+import { LivechatContacts, LivechatRooms, LivechatVisitors } from '@rocket.chat/models';
+
+import { Livechat } from '../../../../../../app/livechat/server/lib/LivechatTyped';
+import { i18n } from '../../../../../../server/lib/i18n';
+
+export async function changeContactBlockStatus({ contactId, block, visitorId }: { contactId: string; visitorId: string; block: boolean }) {
+	const contact = await LivechatContacts.findOneById<Pick<ILivechatContact, '_id' | 'channels'>>(contactId, {
+		projection: { _id: 1, channels: 1 },
```

Comment:
```suggestion
		projection: { channels: 1 },
```

id always comes unless you put _id: 0.

Reply:
noted.